vcpkg.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "minio-cpp",
-  "version": "0.1.1",
+  "version": "0.2.0",
```

Comment:
Is it possible to generate this file or replace this value at the time of build?

Reply:
I have never seen generating `vcpkg.json` file during the build. I cannot even imagine how that would be supposed to work if `vcpkg.json` file needs to be read even before you call cmake (because vcpkg needs vcpkg.json in order to prepare dependencies that will be then used by cmake).

I would rather follow what other projects do in this case.